test/edu/umass/cs/gnsclient/client/singletests/SimpleHttpClientTest.java

```diff
@@ -25,21 +27,21 @@
  * @author westy
  */
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
-public class SimpleHttpClientTest {
+public class SimpleHttpClientTest extends DefaultGNSTest {
 
-  private static final String ACCOUNT_ALIAS = "test@gns.name"; // REPLACE THIS WITH YOUR ACCOUNT ALIAS
-  private static final String PASSWORD = "password";
-
-  private HttpClient client;
+  private HttpClient httpClient;
 
   private static GuidEntry masterGuid;
+  private static GuidEntry httpOneEntry;
+  private static GuidEntry httpTwoEntry;
+
 
   /**
    *
    */
   public SimpleHttpClientTest() {
-    if (client == null) {
-      client = new HttpClient("127.0.0.1", 8080);
+    if (httpClient == null) {
+      httpClient = new HttpClient("127.0.0.1", 24703);
```

Comment:
General question, not a code review comment; Why was this changed?
Since many applications rely on port 8080, I guess a documentation update is necessary so that nothing breaks. The CASA Android developers rely on port 8080, for example.